conda_recipe/build.sh

```diff
@@ -14,7 +14,7 @@ rm -rf build
 mkdir build
 cd build
 if [ -z "${CMAKE_BUILD_PARALLEL_LEVEL}" ]; then
-  export CMAKE_BUILD_PARALLEL_LEVEL="$CPU_COUNT"
+  export CMAKE_BUILD_PARALLEL_LEVEL=1
```

Comment:
Looks like `CPU_COUNT` can be overridden from the environment: https://github.com/conda/conda-build/pull/1149 Can you check if that would work rather than changing `build.sh` itself?

cc @arthurp 

https://github.com/conda/conda-build/pull/1149

Reply:
Does cmake honor `CPU_COUNT`? I vaguely remember finding out that it didn't and you had to set `CMAKE_BUILD_PARALLEL_LEVEL`. I think that's why I wrote this code initially.

Reply:
I'll try that out, but I suspect the env var will get unset at some point like `CMAKE_BUILD_PARALLEL_LEVEL` did in my previous attempt at fixing this: https://github.com/KatanaGraph/katana/pull/846/commits/d375267795f5f7ba5efb725416da10093718e1d0

Reply:
As @ddn0, I think you could set CPU_COUNT in the code that calls conda-build instead of here.

Reply:
`conda-build` unsets most variables, but those issues say it passes through `CPU_COUNT`.